store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9640,4 +9640,26 @@ TODO: delete them permanently from here
   <globalConfigValue>1518163473</globalConfigValue>
 </attr>
 
+<attr id="3024" name="zimbraFeatureResetPassword" type="boolean" cardinality="single" optionalIn="account,cos,domain" flags="accountCosDomainInherited" since="8.8.9">
+  <defaultCOSValue>FALSE</defaultCOSValue>
+  <desc>whether or not password reset feature is enabled</desc>
+</attr>
+
+<attr id="3025" name="zimbraPrefRecoveryAddress" type="cs_emailp" cardinality="single" optionalIn="account" callback="PrefMailForwardingAddress" since="8.8.9">
```

Comment:
Make it zimbraPrefPasswordRecoveryAddress. Also we do not need the PrefMailForwardingAddress attribute

Reply:
Done